README.md

```diff
@@ -260,6 +260,39 @@ npm run code
 npm test
 ```
 
+## Plugins
+
+Sinon chrome module supports by plugins.
+
+### Cookie plugin
+
+You can test chrome cookie behaviour.
+
+#### install
+
+```js
+var chrome = require('sinon-chrome');
+var cookiePlugin = require('sinon-chrome/out/plugins/cookies');
```

Comment:
1. Plugins already located in `src/plugins` (but es6)
2. I think about it.
3. I think, the best way is configure chrome object by `create` method, that should have ability to inject plugins.

```js
var create = require('sinon-chrome').create;
var chrome = create(pluginList);
```

`require('sinon-chrome')` and `create([])` should returns pure chrome stub object for backward compatibility.